### PR TITLE
chore: bump sdk to v0.23.0

### PR DESCRIPTION
**Pull request by classicalliu**
1. bump SDK to v0.23.0
2. split `sendCapacity` to `sendTx` and `generateTx`
3. add `calculateFee`
4. support generate transaction with `feeRate`
5. set default fee rate to 1000 if fee not set in wallets controller `sendCapacity`

### CODE REVIEW

**Comment by ashchan** [comment]
Too quick to click approve button. Sorry.

**Comment by Keith-CY** [comment]
ckb-sdk-js@v0.23.1 has been published

File: packages/neuron-wallet/src/controllers/wallets.ts [hunk 1]
```diff
@@ -348,35 +349,91 @@ export default class WalletsController {
...
         params.items,
-        params.password,
         params.fee,
-        params.description
+        params.feeRate,
       )
       return {
         status: ResponseCode.Success,
-        result: hash,
+        result: JSON.stringify(tx),
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/controllers/wallets.ts]: Why use `stringify` here?

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/wallets.ts]: Just a serialization here, can do it or not.

Comment (ashchan) [reply packages/neuron-wallet/src/controllers/wallets.ts]: Let’s do that only to the api controller later.

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/wallets.ts]: Removed now.

File: packages/neuron-wallet/src/services/cells.ts [hunk 2]
```diff
@@ -143,7 +161,22 @@ export default class CellsService {
     return {
       inputs,
       capacities: inputCapacities.toString(),
+      needFee: needFee.toString(),
+    }
+  }
+
+  public static everyInputFee = (feeRate: bigint): bigint => {
+    /*
+    * every input need 44 Bytes
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/cells.ts]:
```suggestion
    * every input needs 44 Bytes
```

File: packages/neuron-wallet/src/services/cells.ts [hunk 3]
```diff
@@ -143,7 +161,22 @@ export default class CellsService {
...
+    * every input need 1 witness signed by secp256k1, with 65 Bytes data, serialized in 69 Bytes, add extra 4 Bytes when add to transaction.
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/cells.ts]:
```suggestion
    * every input needs 1 witness signed by secp256k1, with 65 Bytes data, serialized in 69 Bytes, add extra 4 Bytes when add to transaction.
```

**Pull request merged**